nsls2ptycho/core/databroker_api.py

```diff
@@ -22,7 +22,10 @@
 
 
 def _load_HXN():
-    import nsls2ptycho.core.HXN_databroker as hxn_databroker
+    try:
+        import nsls2ptycho.core.HXN_databroker as hxn_databroker
+    except ImportError:
+        raise RuntimeError("[WARNING] Databroker is not found and so disabled.")
```

Comment:
I think it should have a type of "Error" as it's rising. Same below.
```suggestion
        raise RuntimeError("[Error] Databroker is not found and so disabled.")
```

Reply:
Thanks Maksim! It's better be a warning I think, because this "error" is immediately caught by the outer try-except and simply printed (at the start time of GUI):
https://github.com/NSLS-II/ptycho_gui/blob/dbb3c8799edbddcf91275399438908b57f4cdd45/nsls2ptycho/core/databroker_api.py#L73-L74
Databroker is not a mandatory dependency after all.